src/core/components/hv-screen/index.js

```diff
@@ -218,6 +218,17 @@ export default class HvScreen extends React.Component {
       }
       const stylesheets = Stylesheets.createStylesheets(doc);
       this.navigation.setRouteKey(this.state.url, routeKey);
+
+      // Clean the preload screen out of cache after loading the screen
```

Comment:
is this redundant with the clean up done in hv-route, or complimentary?

Reply:
This is because the thing which gets rendered may not be an `hv-screen`, it could be a `<navigator>`